### PR TITLE
implements comment_count

### PR DESCRIPTION
**Pull request by steelfish1010**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -46,7 +60,7 @@ describe("GET /api/article/:article_id", () => {
   });
 });
```
Comment (xmenbteam) [inline __tests__/app.test.js:61]: How about a 400: article_id = 'cheese'?

**Review by xmenbteam — APPROVED** [review]
great!

File: __tests__/app.test.js (hunk 1 above)
Comment (steelfish1010) [reply __tests__/app.test.js:61]: is my article_id = hello not cheesy enough? :)

Comment (steelfish1010) [reply __tests__/app.test.js:61]: OK, you win!

Comment (xmenbteam) [reply __tests__/app.test.js:61]: Lol sorry!

**Review by xmenbteam — APPROVED** [review]

**Pull request merged**